unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/PostWebRequestFactory.cs

```diff
@@ -9,7 +10,13 @@ public class PostWebRequestFactory : IPostWebRequestFactory
         public void SetBody(string data) =>
             this.postData = data;
 
-        public UnityWebRequest CreateWebRequest(string url) =>
-            UnityWebRequest.Post(url, postData);
+        public UnityWebRequest CreateWebRequest(string url)
+        {
+            var unityWebRequest = new UnityWebRequest(url, "POST");
```

Comment:
What is the difference between this and what was before?

Reply:
We need to send all jsons encoded in utf-8, if we don't do that we get a 400 - bad request